.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.6
```

Comment:
Could we target Ruby 2.5, assuming that we want to test it on Travis? (see my other comment)